packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js

```diff
@@ -1,23 +1,26 @@
 import { DKIM_CONFIGURATION_GUIDE } from './emailpro-domain-dkim-autoconfig.constants';
 
-export default class EmailProDomainDkimAutoconfigCtrl {
+import DkimAutoConfigurator from './dkim-auto-configurator';
+
+export default class EmailProDomainDkimAutoconfigCtrl extends DkimAutoConfigurator {
```

Comment:
Why do you split your controller in 2 classes whereas you don't use it anywhere else ?

Reply:
I wanted to extract the logical part of dkim autoconfiguration which is the same between emailpro and exchange.

I kept it in each corresponding module